### ISSUE TITLE
PSM status after usage payment (LP) (Ready for UAT on PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#11422]
### When the PSM folder status is 'Pending Payment' and the ROW Usage Fee(s) are paid, the system should update the folder status to "Pending Permit". 

_Currently the folder status stays in 'Pending Payment' after the payment is made. This is incorrect and could cause confusion with customers and reviewers._

----
**Acceptance Criteria:** 

- [x] Folder type PSM, no subtype exception
- [x] When ROW usage fees are paid the system should check folder status
- [x] If the folder status is Pending Payment, the system should update the folder status to Pending Permit

**Comment by TracyLinder** [comment]
This ticket wasn't requested by a customer; I noticed this while testing the PSM folder. We need to discuss this with ROW before implementation to be sure they agree this should be done and approve the AC.

**Comment by TracyLinder** [comment]
I did speak w/ ROW about this and they agreed it's a problem and should be fixed as proposed in the AC

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Committed changes and requested for Dev and Test promotion.
SCTASK0349742

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT this has been deployed. Please UAT

**Comment by TracyLinder** [comment]
### Update
- Priya tested in TEST but looks like not working. When she pays the fee, the status goes to Internet Pending. 

**Answer**
1. If the fee paid is an application fee, the status should update to Internet Pending & Web Application Process should insert (others might also, not sure)
2. If the fee paid is a ROW usage fee, the status should update to Pending Permit

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Done the commit and requested for Dev/Test deployment
SCTASK0354927

**Comment by Nadin-Nader** [comment]
Sent Message to Kyle asking for him to UAT this on TEST. Waiting for response

**Comment by KyleErich** [comment]
@Nadin-Nader @Lpriya-LIT UAT on TEST approved. Thank you.

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT Kyle has approved this to go to PROD.

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Created Prod Deployment request
SCTASK0361102

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT  this has been deployed to PROD and i asked Kyle to UAT it. Waiting on Response

**Comment by Nadin-Nader** [comment]
Per Kyle:

> I was unable to UAT this since the AB+C system would not accept my test payment using the fake credit card number. I guess we can approve since it worked on Test, unless anyone knows a way to successfully UAT this.

@KyleErich  If there are issues with it on PROD we can open another card up or reopen this one.
